README.md

```diff
@@ -40,109 +116,222 @@ You can download the script from the following locations:
 
         wget https://github.com/sandain/MinecraftServerControlScript/archive/master.zip
 
-* Make a clone of the [git repository](https://github.com/sandain/MinecraftServerControlScript.git):
 
-        git clone https://github.com/sandain/MinecraftServerControlScript.git
 
 ### Configuration
-To get your server to run the script on startup, and cleanly down the server
-on shutdown, the `mscs` script must be copied to `/usr/local/bin/`,
-have its execute permissions set, and the system must be instructed to use
-the script on startup and shutdown.  For Bash programmable completion
-support, the `mscs.completion` script must be copied to
-`/etc/bash_completion.d/`.  For security reasons, the script uses a user
-account named `minecraft` rather than `root` and the account must be created
-before the script is used.
-
-This can all be done automatically with the included Makefile in Debian and
+
+Navigate to the `MinecraftServerControlScript` directory that you just downloaded. Configuration can be done with the included Makefile in Debian and
 Ubuntu like environments by running:
```

Comment:
This looks like you are exceeding the 80 character limit on this line.  I would like the README to be readable from a command line, and more than 80 chars makes this difficult.